example/ck_tile/01_fmha/codegen/ops/fmha_fwd_splitkv.py

```diff
@@ -574,6 +637,9 @@ def get_pipelines(dtype, hdim) -> List[FmhaFwdSplitKVPipeline]:
                     if pipeline.F_spad != 't' or pipeline.F_skpad != 't':
                         # in group mode, spad/skpad must be true, since we can't predict if seqlen of current batch need pad or not
                         continue
+                    if pipeline.F_pagedkv == 't':
```

Comment:
Hi, Why should we disbale the code generation for grouped paged kv if group and batch mode's only difference is offset of `q`?